db/repos/rounds.js

```diff
@@ -68,7 +68,7 @@ class RoundsRepository {
 	updateMissedBlocks (backwards, outsiders) {
 		return this.db.none(sql.updateMissedBlocks, {
 			change: backwards ? '- 1' : '+ 1',
-			outsiders
+			outsiders: outsiders
```

Comment:
This is not needed in ES6. In fact, the syntax that I used is the one recommended everywhere ;)